CppSamples/EditData/CreateKMLMultiTrack/README.md

```diff
@@ -0,0 +1,38 @@
+# Create KML multi-track
+
+Create, save and preview a KML multi-track, captured from a location data source.
+
+![Create KML multi-track](screenshot.png)
+
+## Use case
+
+When capturing location data for outdoor activities such as hiking or skiing, it can be useful to record and share your path. This sample demonstrates how you can collect individual KML tracks during a navigation session, then combine and export them as a KML multi-track.
+
+## How to use the sample
+
+Tap **Start Navigation** to begin moving along a simulated trail. Tap **Record Track** to start recording your current path. Tap **Stop Recording** to end recording and capture a KML track. Repeat these steps to capture multiple KML tracks in a single session. Tap the **Save** button to save your recorded tracks as a `.kmz` file to local storage. Then load the created `.kmz` file containing your KML multi-track to view all created tracks on the map.
+
+## How it works
+
+1. Create an `ArcGISMap` with a basemap and a `GraphicsOverlay` to display the path geometry for your navigation route.
```

Comment:
Sometimes the common descriptions need to be modified as type names can vary slightly 
```suggestion
1. Create a `Map` with a basemap and a `GraphicsOverlay` to display the path geometry for your navigation route.
```